rules/linux/lnx_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,28 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    category: process_creation
+    product: unix
+detection:
+    selection:
+        CommandLine|contains:
+            # Linux Only
+            - 'arp -a'
```

Comment:
could you please tell me how arp table reflects firewall configuration?

Reply:
It does not, I'll be removing that in the next commit. Thanks!

Reply:
On further review, I think I had tunnel vision focusing on firewall enumeration. 

T1016 is more generally network configuration enumeration, not limited to the firewall. So in this case, I think I should rename the rule to drop "Firewall Enumeration" and simply catch as many network discovery commands as reasonable. The coming commit should reflect that.